src/graphql/typeDefs.graphql

```diff
@@ -13,6 +13,9 @@ type Query {
     first: Int
     orderBy: UserArticleLinksOrderBy
   ): UserArticleLinkConnection @auth(check: ["userId"])
+
+  # Get user's notification setting
+  setting: UserSetting @auth
```

Comment:
We can check `userId` only using `@auth(check: ["userId"])`

The same goes for `allowNotification` mutation